lib/sales_engine.rb

```diff
@@ -47,4 +47,16 @@ def items_per_merchant
   def average_items_per_merchant_standard_deviation
     @merchants.average_items_per_merchant_standard_deviation
   end
+
+  def number_of_invoices_per_merchant
+    @merchants.number_of_invoices_per_merchant
+  end
+
+  def average_invoices_per_merchant
+    @merchants.average_invoices_per_merchant.round(2)
+  end
+
+  def average_invoices_per_merchant_standard_deviation
+    @merchants.average_invoices_per_merchant_standard_deviation
+  end
```

Comment:
Although we have yet to refactor away some of the helper methods in engine, Alex and I took to simply calling @engine.merchants or @engine.items in cases where the engine does nothing more than forward the message. We found it difficult / unnecessary to test this message forwarding and simply accepted our minor uneasy considering 'Law of Demeter' problems to reduce the bloat of the SE class.